docs/contract-rate-non-revision-design.md

```diff
@@ -0,0 +1,77 @@
+# Contract and Rate Without Revisions Design
+
+This is an ER diagram of the data that is planned to be sent to Salesforce on submission. It relies on Salesforce's native diff tracking so elides the concept of Revisions. 
+
+```mermaid
+erDiagram
+
+Contract {
+    String id
+    DateTime initiallySubmittedAt
+    DateTime updatedAt
+    StateCode stateCode
+    String name
+    String mccrsID
+
+    String[] programNicknames
+    PopulationCoveredEnum populationCovered "MEDICAID, CHIP, MEDICAID_AND_CHIP" 
+    SubmissionTypeEnum submissionType "CONTRACT_ONLY, CONTRACT_AND_RATES"
+    Boolean riskBasedContract
+    String submissionDescription
+    Document[] contractDocuments
+    Document[] supportingDocuments
+    ContractTypeEnum contractType "BASE, AMENDMENT"
+    ContractExecutionStatusEnum contractExecutionStatus "EXECUTED, UNEXECUTED"
+    CalendarDate contractDateStart
+    CalendarDate contractDateEnd
+    String[] managedCareEntities
```

Comment:
Are you assuming for these two we will return the user friendly names as strings rather than the enum values? Trying to understand why these are strings whereas some others are enums

Reply:
I'm going back and forth on how to do enums, these two I know at some point one of them was a proper enum and one wasn't but I guess they both are now GraphQL. I think I'd like to try one and the other to see how Salesforce handles that so we can decide if we care about putting enums into that system